meson.build

```diff
@@ -72,7 +72,6 @@ glib_version = '>=2.45.7'
 math    = cc.find_library('m', required: true)
 
 gtk     = dependency('gtk+-3.0',                    version: '>=3.10.0')
-gtk_wl  = dependency('gtk+-wayland-3.0',            version: '>=3.10.0')
```

Comment:
I didn't see `gtk_wl` being referenced anywhere, so I assumed it didn't serve any purpose. It looks like `nemo` is the only other repo directly referencing `gtk+-wayland-3.0`, and its `gtk_wl` also appears referenced.

I did not see any build/linking errors on my system, but maybe CI/Mint/Debian uses different linker flags? The CI failure doesn't look related to this.

Reply:
Ah you're right, sorry.  Interestingly gtk+-wayland-3.0.pc and gtk+-3.0.pc are identical.

Looks like I added in 5b3f274ec50, I think I assumed it would be required without checking.